plugins/node/opentelemetry-instrumentation-aws-sdk/test/s3.test.ts

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  getTestSpans,
+  registerInstrumentationTesting,
+} from '@opentelemetry/contrib-test-utils';
+import { AwsInstrumentation } from '../src';
+registerInstrumentationTesting(new AwsInstrumentation());
+
+import * as AWS from 'aws-sdk';
```

Comment:
awesome to have tests for that.

Please note that those tests are for v2 which [will soon be end of life](https://aws.amazon.com/blogs/developer/announcing-end-of-support-for-aws-sdk-for-javascript-v2/).
Consider also adding assertions on these new attributes to v3 tests in `test/aws-sdk-v3.test.ts` [here](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/60a99c98fd3a5594c7c2234184f06166b375e707/plugins/node/opentelemetry-instrumentation-aws-sdk/test/aws-sdk-v3.test.ts#L66)